package.json

```diff
@@ -2,6 +2,14 @@
   "name": "web-push-encryption",
   "version": "2.0.0",
   "description": "A library that handles payload encryption for the Web Push protocol",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/web-push-encryption.git"
```

Comment:
@gauntface This is what `npm init` does by default. You can also see a similar usage: `https://github.com/GoogleChrome/push-notifications/blob/master/package.json`